README.md

```diff
@@ -72,9 +70,10 @@ available:
 | `BUILD_COMM_TCP_TLS`   | TRUE \| FALSE | FALSE - UDP is used |
 | `USE_LOG4CPP`          | TRUE \| FALSE | FALSE |
 | `CONCORD_LOGGER_NAME`  | STRING |"concord" |
+| `USE_CONAN`            | OFF\| ON | ON |
 
- Note: You can't set both `BUILD_COMM_TCP_PLAIN` and `BUILD_COMM_TCP_TLS` to TRUE.
- 
+ Note(1): You can't set both `BUILD_COMM_TCP_PLAIN` and `BUILD_COMM_TCP_TLS` to TRUE.
+ Note(2): In case of setting `USE_CONAN=OFF` cmake will search for packages in their native location (`/usr/local`).
```

Comment:
Nit: I wouldn't specify the install location here. It can change, and users can have all sorts of wacky configurations on their systems. They may set PREFIX= to indicate other places, and we don't want to have to update this README to reflect changes in future supported platforms.

Reply:
done